eng/Version.Details.xml

```diff
@@ -1,75 +1,75 @@
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Internal" Version="6.0.0-alpha.1.20468.7" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.NETCore.App.Internal" Version="6.0.0-alpha.1.20477.5" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>a820ca1c4f9cb5892331e2624d3999c39161fe2a</Sha>
+      <Sha>ae1b1be2ba3147758345062f9e78afb8cc2ff39b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.0-alpha.1.20468.7" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="6.0.0-alpha.1.20477.5" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>a820ca1c4f9cb5892331e2624d3999c39161fe2a</Sha>
+      <Sha>ae1b1be2ba3147758345062f9e78afb8cc2ff39b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Host.win-x64" Version="6.0.0-alpha.1.20468.7" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.NETCore.App.Host.win-x64" Version="6.0.0-alpha.1.20477.5" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>a820ca1c4f9cb5892331e2624d3999c39161fe2a</Sha>
+      <Sha>ae1b1be2ba3147758345062f9e78afb8cc2ff39b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.DotNetHostResolver" Version="6.0.0-alpha.1.20468.7" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.NETCore.DotNetHostResolver" Version="6.0.0-alpha.1.20477.5" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>a820ca1c4f9cb5892331e2624d3999c39161fe2a</Sha>
+      <Sha>ae1b1be2ba3147758345062f9e78afb8cc2ff39b</Sha>
     </Dependency>
     <!-- Change blob version in GenerateLayout.targets if this is unpinned to service targeting pack -->
     <!-- No new netstandard.library planned for 3.1 timeframe at this time. -->
     <Dependency Name="NETStandard.Library.Ref" Version="2.1.0" Pinned="true">
       <Uri>https://github.com/dotnet/core-setup</Uri>
       <Sha>7d57652f33493fa022125b7f63aad0d70c52d810</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Platforms" Version="6.0.0-alpha.1.20468.7" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.NETCore.Platforms" Version="6.0.0-alpha.1.20477.5" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>a820ca1c4f9cb5892331e2624d3999c39161fe2a</Sha>
+      <Sha>ae1b1be2ba3147758345062f9e78afb8cc2ff39b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="5.0.0-rc.2.20474.4" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="5.0.0-rc.2.20466.8" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
@mmitche we have a downgrade coming in here along with the SDK insertion

Reply:
This is the version that that dotnet/sdk has right now (which is why it downgrades). It's actually not a downgrade, probably. aspnetcore is producing rc.2 branded items out of both master and the rc2 branch.

Reply:
Gotcha, the wrong version of System.IO.Pipelines seems to be included as well, I'm not sure where this one is coming from

> Error: Could not load file or assembly 'System.IO.Pipelines, Version=4.0.0.1, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified.

Reply:
@mmitche I can't find where the wrong version of the System.IO.Pipelines dependency is coming from, do you happen to know?